server/datastore/mysql/mysql.go

```diff
@@ -107,10 +107,10 @@ func (ds *Datastore) loadOrPrepareStmt(ctx context.Context, query string) *sqlx.
 	return stmt
 }
 
-// NewMDMAppleSCEPDepot returns a *apple_mdm.MySQLDepot that uses the Datastore
+// NewMDMAppleSCEPDepot returns a *mysql.SCEPDepot that uses the Datastore
 // underlying MySQL writer *sql.DB.
-func (ds *Datastore) NewMDMAppleSCEPDepot(caCertPEM []byte, caKeyPEM []byte) (*apple_mdm.SCEPMySQLDepot, error) {
-	depot, err := apple_mdm.NewSCEPMySQLDepot(ds.writer.DB, caCertPEM, caKeyPEM)
+func (ds *Datastore) NewSCEPDepot(caCertPEM []byte, caKeyPEM []byte) (scep_depot.Depot, error) {
+	depot, err := newSCEPDepot(ds.writer.DB, caCertPEM, caKeyPEM)
```

Comment:
No big deal but might as well do:
```suggestion
	return newSCEPDepot(ds.writer.DB, caCertPEM, caKeyPEM)
```

Reply:
agreed! fixed.